lib/bolt/node/winrm.rb

```diff
@@ -245,5 +251,19 @@ def _run_task(task, input_method, arguments)
         end
       end
     end
+
+    def escape_arguments(*arguments)
+      arguments.map do |arg|
+        if arg =~ /^['].*[']$/
+          "\"#{arg}\""
+        elsif arg =~ /^["].*["]$/
+          "'#{arg}'"
+        elsif arg =~ / /
+          "\"#{arg}\""
+        else
+          arg
+        end
+      end.join(' ')
```

Comment:
Ideally I'd like a Shellwords.escape for powershell. Something like https://github.com/larskanis/shellwords/tree/master/lib/shellwords

Reply:
Or https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/

Reply:
Yeah, I've read that blog post a few times... it's a good one ;0